### DISCUSSION TITLE
in chrome browser some emoji are not shown and have no color

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1859] [General]
in chrome browser some emoji are not shown. can this be fixed for the chrome browser?
![image](https://user-images.githubusercontent.com/13809377/208677607-d45deff3-6030-4c7e-aeed-6cc5e0cf99f1.png)

in firefox they are all shown with colour:
![image](https://user-images.githubusercontent.com/13809377/208678032-ec44d29d-036f-476c-9194-787e168cad5b.png)

**Comment by remdex** [comment]
That's your PC issue on windows all icons are shown just fine :)
